src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -29,18 +33,25 @@ internal sealed class AspNetCoreDiagnosticObserver : DiagnosticObserver
 
         private const string DiagnosticListenerName = "Microsoft.AspNetCore";
         private const string HttpRequestInOperationName = "aspnet_core.request";
+        private const string MvcOperationName = "aspnet_core.mvc";
```

Comment:
I think we want something like `aspnet_core_mvc.request` here. I think the `.request` suffix is a convention on all incoming http requests.

Reply:
Separate issue maybe for a separate PR: some operation names have hyphens (`aspnet-mvc.request`) and others underscores (`aspnet_core.request`), but I think they all normalized into one or the other, so it's probably safe to fix them in code without breaking changes.

Reply:
Renamed `aspnet_core.mvc` -> `aspnet_core_mvc.request` in https://github.com/DataDog/dd-trace-dotnet/pull/1289/commits/bd6744cc75b177a49d182462d087d2b7aff05604